src/less/variables/base/menu-variables.less

```diff
@@ -1 +1,15 @@
-@menu-item-disabled-foreground-color: @color-text-disabled;
+@menu-background-color: @selection-list-background-color;
```

Comment:
Is there a reason why we need to define all these variables again and just point them to `selection-list`? Why not use `selection-list` directly?
Seems a bit overkill to redefine all these variables. I can see it for organization, but I think it gets confusing what the original color when you have like 3 levels of variable definitions (base color > selection-list > menuitem/listbox-option)

I think the only time we should define a variable is when one of these components needs it's own color different than the others. But for now if we add a new color or something for all dropdowns we have to maintain it in like 3+ different places
I think calling it `item` directly makes it just as intuitive and generic for both listbox and menu.

Reply:
Yeah, that's a fair comment and good observation. I did have it that way originally - just pointing to the `selection-list-item` variables directly as you say. However, I felt it might not be too obvious what a 'selection-list' is when looking through the code, and so terms like `menu-menuitem` and `listbox-option` seem to hammer home the difference.

If we add a new color, we shouldn't have to update it in 3 places though, so I'm not sure what you mean. We would only need to change the colour at the `selection-list` level, right? If you are correct, then I will consider reverting, because that wouldn't be good.